docs/releases/1.22-NOTES.md

```diff
@@ -18,6 +18,8 @@ This is a document to gather the release notes prior to the release.
 
 * Support for Kubernetes version 1.17 is deprecated and will be removed in kOps 1.23.
 
+* The legacy location for downloads `s3://https://kubeupv2.s3.amazonaws.com/kops/` has been deprecated and will not be used for new releases. The new canonical downloads location is `https://artifacts.k8s.io/binaries/kops/`.
```

Comment:
Add to 1.21-NOTES.md. For 1.22-NOTES.md, put this in the "Breaking changes" section.

Reply:
Done.